modules/ycc_vm.py

```diff
@@ -540,7 +540,7 @@ def _translate(self):
             if key in ["memory", "disk_size"]:
                 params[key] = params[key] * 2 ** 30
             elif key == "disk_type":
-                params[key] = getattr(DiskType, params[key].upper()).value
+                params[key] = getattr(DiskType, params[key].upper().replace('-','_')).value
```

Comment:
This line should not change ?
https://github.com/arenadata/ansible-module-yandex-cloud/blob/802912567fdb7ff428ecddf9a54103a439d9b3ed/modules/ycc_vm.py#L549